### PR TITLE
Add District09 plugins to the marketplace

### PR DESCRIPTION
**Pull request by Jelle-S**

### CODE REVIEW

File: _store/district09-archive-timesheets-command-bundle.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+github: "https://github.com/digipolisgent/kimai_plugin_archive-timesheets-command"
+type: plugin
+tags:
+  - plugin
+bundle:
+  name: "ArchiveTimesheetsCommandBundle"
+  clone: "https://github.com/digipolisgent/kimai_plugin_archive-timesheets-command.git"
+---
+
+A Kimai 2 plugin, which allows you to archive timesheets older than a specified timeframe, using a command.
```
Comment (kevinpapst) [inline _store/district09-archive-timesheets-command-bundle.md:19]:
```suggestion
A Kimai plugin, which allows you to archive timesheets older than a specified timeframe, using a command.
```

File: _store/district09-archive-timesheets-command-bundle.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
```
Comment (kevinpapst) [inline _store/district09-archive-timesheets-command-bundle.md:20]: Could you please add a sentence what "archiving timesheets" really means?

Archiving sounds as if you can still retrieve them from an archive, but in reality they are deleted.
Not that users expect a real archive and suddenly  experience a data loss.

File: _store/district09-personal-calendar-pref-bundle.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+github: "https://github.com/digipolisgent/kimai_plugin_personal-calendar-pref"
+type: plugin
+tags:
+  - plugin
+bundle:
+  name: "PersonalCalendarPrefBundle"
+  clone: "https://github.com/digipolisgent/kimai_plugin_personal-calendar-pref.git" 
+---
+
+A Kimai 2 plugin, which allows users to set a customized start and end time for
```
Comment (kevinpapst) [inline _store/district09-personal-calendar-pref-bundle.md:19]:
```suggestion
A Kimai plugin, which allows users to set a customized start and end time for
```

**Review by kevinpapst — CHANGES_REQUESTED** [review]
Thanks @Jelle-S 👍 very much appreciated, just some minor comments from my side.

**Review by kevinpapst — APPROVED** [review]

**Comment by kevinpapst** [comment]
Thank you @Jelle-S 🎉

**Comment by Jelle-S** [comment]
And thank _you_!

**Pull request merged**